elastic/integrations
# [Microsoft SqlServer] Update description - Domain users use manual encoding to handle parse error.

- Enhancement

## What does this PR do?

This issue is described here #6064

Added description with an example on how to connect if the domain user is trying to connect with microsoftSqlServer.

## How to test this PR ?

- Install the MicrosoftSqlServer integrations and verify the data collection using AD domain account. 
- User should be able collect metrics by using manual encoding.
- Execute system and pipeline tests.
